CMakeModules/MbedTLS.cmake

```diff
@@ -65,11 +67,17 @@ else()
     if (MbedTLS_FOUND)
         message(STATUS "Found MbedTLS package ${MbedTLS_FOUND}")
     else()
+        if (AVM_STATIC_MBEDTLS)
+            set(MBEDCRYPTO_STATIC "libmbedcrypto.a")
+            set(MBEDX509_STATIC "libmbedx509.a")
+            set(MBEDTLS_STATIC "libmbedtls.a")
+        endif()
+
         include(CheckSymbolExists)
         check_symbol_exists(MBEDTLS_VERSION_NUMBER "mbedtls/version.h" HAVE_MBEDTLS_VERSION_NUMBER)
-        find_library(MBEDCRYPTO mbedcrypto)
-        find_library(MBEDX509 mbedx509)
-        find_library(MBEDTLS mbedtls)
+        find_library(MBEDCRYPTO NAMES ${MBEDCRYPTO_STATIC} mbedcrypto)
```

Comment:
Does it work with avm_static_mbedtls is not set and these variables are not set?